package demo.eventservice.client;

import demo.eventservice.dto.OrganizerDTO;
import org.springframework.cloud.openfeign.FeignClient;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;

@FeignClient(name = "organizer-service")
public interface OrganizerClient {
    @GetMapping("/api/organizers/{id}")
    ResponseEntity<OrganizerDTO> getOrganizerById(@PathVariable("id") Long id);


}
